config.def.h

```diff
@@ -151,7 +151,7 @@ static const char *instantrepeatcmd[] = {"instantrepeat", NULL};
 static const char *instantpacmancmd[] = {"instantpacman", NULL};
 static const char *instantsharecmd[] = {"instantshare", "snap", NULL};
 static const char *nautiluscmd[] = {".config/instantos/default/filemanager", NULL};
-static const char *slockcmd[] = {"ilock", NULL};
+static const char *slockcmd[] = {"sh","-c", "eval $(iconf lockscreen)", NULL};
```

Comment:
```suggestion
static const char *slockcmd[] = {".config/instantos/default/lockscreen", NULL};
```

Reply:
I cannot get to work it this way, settings app doesn't even write to that file nor does it want to execute from the WM keybind

Reply:
That is a separate issue then. It should use this syntax, see the other uses of `~/.config/instantos/devault/` in `config.def.h`.

Reply:
actually my bad, my `defaultapps` script wasn't up to date from the instantOS repo, it actually works now

Reply:
The settings app actually works now, I guess it could be merged